tests/jballerina-unit-test/src/test/resources/test-src/statements/variabledef/record-variable-definition-stmt.bal

```diff
@@ -743,6 +743,47 @@ function testReadOnlyRecordWithMappingBindingPatternInVarDecl() {
     assertEquality(<ReadOnlyRecord> {x: [1, 2], y: "s1"}, b1);
 }
 
+function recordVariableWithAnonymousType() returns [string, boolean] {
+    record {|
+        string name;
+        boolean married;
+    |} {name, married} = checkpanic getPersonRecord().ensureType();
+    return [name, married];
+}
```

Comment:
We don't need separate functions. right? Can be within the test functions itself?

Reply:
Updated